docs/en/latest/router-radixtree.md

```diff
@@ -75,7 +75,7 @@ Here are the rules:
 
 When different routes have the same uri, you can set the priority field of the route to determine which route to match first, or add other matching rules to distinguish different routes.
 
-Note: In matching rules, the `priority` field takes precedence over other rules. Therefore, we can set other matching rules only when the value of the `priority` field is the same.
+Note: In matching rules, the `priority` field takes precedence over other rules.
```

Comment:
We should mention that the `other rules` don't include the `uri` rule.

Reply:
Updated.